var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -39,9 +39,8 @@ class Elfutils(AutotoolsPackage):
     depends_on('automake', type='build')
     depends_on('autoconf', type='build')
```

Comment:
Now that you're fetching a release tarball instead of git source code, you can remove these dependencies. They are only needed to generate a configure script, but this tarball already comes with one.